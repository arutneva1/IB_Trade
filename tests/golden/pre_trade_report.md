NetLiq: 100000.00
Cash USD: 10000.00
Cash Buffer: 5000.00

| symbol | target_pct | current_pct | drift_bps | price | dollar_delta | share_delta | side | est_notional | reason |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| AAA | 60.00 | 50.00 | 1000.00 | 100.00 | 10000.00 | 100.0000 | BUY | 10000.00 |  |
| BBB | 40.00 | 50.00 | -1000.00 | 80.00 | -10000.00 | -125.0000 | SELL | -10000.00 |  |
| TOTAL | 100.00 | 100.00 | 0.00 |  | 0.00 |  |  | 0.00 |  |
