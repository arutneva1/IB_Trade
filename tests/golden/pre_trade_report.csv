NetLiq,100000.00
Cash USD,10000.00
Cash Buffer,5000.00

symbol,target_pct,current_pct,drift_bps,price,dollar_delta,share_delta,side,est_notional
AAA,60.0,50.0,1000.0,100.0,10000.0,100.0,BUY,10000.0
BBB,40.0,50.0,-1000.0,80.0,-10000.0,-125.0,SELL,-10000.0
TOTAL,100.0,100.0,0.0,,0.0,,,0.0
